### DISCUSSION TITLE
How to see the webpack warnings?

### DISCUSSION THREAD

**Discussion by gpbl** [facebook/docusaurus#7417] [Q&A] [answered]
When I run `docusaurus start` there are some warnings from webpack:

```
[SUCCESS] Docusaurus website is running at http://localhost:3000/.

✔ Client
  Compiled successfully in 10.58s

client (webpack 5.72.0) compiled with 2 warnings
```

Why it says "compiled with 2 warnings"? How can i see the content of these warnings? Thanks!

**Comment by Josh-Cena** [comment, marked as answer]
You can open the browser and they should be in the browser console.

**Reply by gpbl** [reply]
Awesome thansk @Josh-Cena
